<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GrowlSettings</class>
 <widget class="QWidget" name="GrowlSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="NotifyEnabled">
      <property name="text">
       <string>Enabled</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="StatusChange">
      <property name="text">
       <string>Notify about status changes</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ShowOffline">
      <property name="text">
       <string>Notify when user came offline</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ShowOnline">
      <property name="text">
       <string>Notify when user came online</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Typing">
      <property name="text">
       <string>Notify when contact is typing</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="MessageRecieved">
      <property name="text">
       <string>Notify when message is recieved</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Birthday">
      <property name="text">
       <string>Notify about birthdays</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Blocked">
      <property name="text">
       <string>Notify when message is blocked</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Custom">
      <property name="text">
       <string>Notify about custom requests</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
